### PR TITLE
Floating vl3 autoscale

### PR DESCRIPTION
**Pull request by glazychev-art**
Issue: https://github.com/networkservicemesh/deployments-k8s/issues/6345

### CODE REVIEW

File: examples/floating_interdomain/usecases/FloatingVl3_autoscale/README.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nse-vl3-vpp-1
+spec:
+  containers:
+    - name: nse
+      env:
+        - name: NSM_PREFIXSERVERURL
+          value: "tcp://$vl3_ipam_ip:5006"
```
Comment (denis-tingaikin) [inline examples/floating_interdomain/usecases/FloatingVl3_autoscale/README.md:75]: We definitely shouldn't use static ips. 
See at comment https://github.com/networkservicemesh/deployments-k8s/pull/7054#discussion_r945282488

Comment (glazychev-art) [reply examples/floating_interdomain/usecases/FloatingVl3_autoscale/README.md]: Done

File: examples/floating_interdomain/usecases/FloatingVl3_autoscale/README.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+      env:
+        - name: NSM_PREFIX_SERVER_URL
+          value: "tcp://vl3-ipam.nsm-system.my.cluster3:5006"
+EOF
+```
+
+Save `pod-template.yaml`:
+```bash
+kubectl kustomize -o 'pod-template.yaml'
+```
```
Comment (denis-tingaikin) [inline examples/floating_interdomain/usecases/FloatingVl3_autoscale/README.md:76]: This is looking as a complicaiton. 


Could you please explain why do we need to use this?

**Review by denis-tingaikin — COMMENTED** [review]
Please resolve comment related to `kubect kustomize` .

And also do fixes with dupl of 'Node A' labels on the diagram.  examples/floating_interdomain/usecases/FloatingVl3_autoscale/floating_vl3_autoscale.svg

Also it would be nice to add the source of the diargam into the example that allow modify  the diagram in the future.

File: examples/floating_interdomain/usecases/FloatingVl3_autoscale/README.md (hunk 2 above)
Comment (glazychev-art) [reply examples/floating_interdomain/usecases/FloatingVl3_autoscale/README.md]: You are right. 
Fixed

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**